x-pack/plugins/cases/server/authorization/index.test.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isWriteOperation, Operations } from '.';
+import { OperationDetails } from './types';
+
+describe('index tests', () => {
+  it('should identify a write operation', () => {
+    expect(isWriteOperation(Operations.createCase)).toBeTruthy();
+  });
+
+  it('should identify a read operation', () => {
```

Comment:
nit: `should identify a read operation`. That's the purpose of the this test, right?

Reply:
yeah good point, I guess it doesn't necessarily mean that it is a `read operation` just that it's not a `write operation` 😆  I'll switch it to say `should return false when the operation is not a write operation`